configure

```diff
@@ -1,5 +1,23 @@
 #!/usr/bin/env bash
 
+set -e
+set -o pipefail
+
+function usage {
+  echo "Usage configure [-d]"
+  echo "-d Use default configuration settings."
```

Comment:
Instead of adding a new flag, we could just run `yes "" | ./configured" in the CI build, right?